core/common/src/main/java/alluxio/util/ConfigurationUtils.java

```diff
@@ -38,73 +39,140 @@
   private ConfigurationUtils() {} // prevent instantiation
 
   /**
+   * Reads a property list (key and element pairs) from the input byte stream. The input stream is
+   * in a simple line-oriented format and is assumed to use the ISO 8859-1 character encoding; that
+   * is each byte is one Latin1 character. Characters not in Latin1, and certain special characters,
+   * are represented in keys and elements using Unicode escapes as defined in section 3.3 of
+   * <cite>The Java&trade; Language Specification</cite>. The specified stream remains open after
+   * this method returns.
+   *
+   * @param inStream the input stream
+   * @param destination the destination map to store the result
+   * @exception IOException if an error occurred when reading from the input stream
+   * @throws IllegalArgumentException if the input stream contains a malformed Unicode escape
+   *         sequence
+   */
+  public static void load(InputStream inStream, ConcurrentHashMapV8<String, String> destination)
```

Comment:
Instead of `ConcurrentHashMapV8<String, String>`, how about making it `Map<String, String>` to be more general